core/windows_and_tabs/window_management.talon

```diff
@@ -17,8 +17,7 @@ snap last [screen]: user.move_window_previous_screen()
 snap screen <number>: user.move_window_to_screen(number)
 snap <user.running_applications> <user.window_snap_position>:
     user.snap_app(running_applications, window_snap_position)
-# <user.running_applications> is here twice to require at least two applications.
-snap <user.window_split_position> <user.running_applications> <user.running_applications>+:
-    user.snap_layout(window_split_position, running_applications_list)
+layout <user.window_layout>: user.snap_layout(window_layout)
```

Comment:
I really like this functionality. This is an example of a complicated grammar that might benefit from a little documentation. It's a little hard to quickly piece together what can be done just by looking at the python or talon files. 

Maybe a small readme makes sense for this functionality or a few comments in this file?

Reply:
Sounds good. I added to read me describing how to use it.